rules/windows/defense_evasion_iis_httplogging_disabled.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/04/14"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+An attacker with IIS server access via a webshell or alike can disable HttpLogging as an effective anti-forensics
+measure. Verify the legitimacy context of this change.
```

Comment:
```suggestion
Identifies when Internet Information Services (IIS) HTTP Logging is disabled. An attacker with IIS server access via a webshell or other mechanism can disable HTTP Logging as an effective anti-forensics measure.
```

Reply:
If you know what level of access is required to disable HTTP Logging on an IIS server, can we put that in the description of the rule too?

I'd recommend moving the sentence below to the notes field of the rule, as the description field is typically used to describe the behavior that the rule detects and the motive for why an attacker might execute the technique. The notes field contains information that's helpful for analysts to triage/investigate the alert.

> Verify the legitimacy context of this change.